client/src/Movies/MovieList.js

```diff
@@ -34,21 +38,11 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
-    </div>
+    <MovieCard
+      title={title}
+      director={director}
+      metascore={metascore}
+      stars={stars}
```

Comment:
This is fine, personally I think it looks clean passing the `movie` object and destructuring on the component side